From 3e196e08929ac9ade0bd54077bd8fdb6b6815a7c Mon Sep 17 00:00:00 2001
From: Lin Fuzhen <fuzhen.lin@freescale.com>
Date: Fri, 24 Aug 2012 13:00:29 +0800
Subject: [PATCH 007/115] ENGR00221314-1 Profile: remove some debug message
 and refine code

refine some code and cleanup  debug message

Signed-off-by: Lin Fuzhen <fuzhen.lin@freescale.com>
---
 .../src/fsl/power/service/FSLPowerOptionService.java             | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

diff --git a/FSLProfileService/src/fsl/power/service/FSLPowerOptionService.java b/FSLProfileService/src/fsl/power/service/FSLPowerOptionService.java
index a3c3121..5081356 100755
--- a/FSLProfileService/src/fsl/power/service/FSLPowerOptionService.java
+++ b/FSLProfileService/src/fsl/power/service/FSLPowerOptionService.java
@@ -131,17 +131,14 @@ public class FSLPowerOptionService extends Service{
 	//showNotification();
 	//testProvider();
 	String sys_rev = get_system_rev();
-	byte rev[] = null;
 	if(!sys_rev.isEmpty()){
-		//rev = sys_rev.getBytes(US-ASCII);
-			gSocType = (int) Long.parseLong(sys_rev.substring(1, 3));
-			Log.i(TAG, "get soc type " + gSocType +"string:"+sys_rev.substring(1,3));
+		gSocType = (int) Long.parseLong(sys_rev.substring(1, 3));
+		Log.i(TAG, "get soc type " + gSocType +"string:"+sys_rev.substring(1,3));
 	}
 	else
 		Log.e(TAG, "could not get SOC type");
 
 	if(!sys_rev.isEmpty()){
-		//rev = sys_rev.getBytes(US-ASCII);
 		gSocRev = (int) Long.parseLong(sys_rev.substring(4));
 		if(DEBUG) Log.i(TAG, "get soc revision " + gSocRev +"string:"+sys_rev.substring(4));
 	}
@@ -276,7 +273,7 @@ public class FSLPowerOptionService extends Service{
 						 new FileInputStream( "/proc/cpuinfo") ), 1000 );
 				version = reader.readLine();
 				while (version != null){
-					Log.i(TAG,"get line: " +version);
+					if(DEBUG) Log.i(TAG,"get line: " +version);
 					version = reader.readLine();
 					if (version.startsWith("Revision")){
 						tmp = version.split(":");
-- 
1.8.0

